spec/services/webhook_manager/donation_creator_spec.rb

```diff
@@ -17,35 +17,6 @@
     expect(item.item_type).to eq('donation')
     expect(donation.item).to eq(item)
     expect(donation.amount).to eq(payload[:amount])
-    expect(PaymentIntent.last.successful).to be true
```

Comment:
we don't need these tests anymore since they're handled by the webhook controller. in the logic, if a payment intent is successful we raise an error